pillarbox-ui/src/main/java/ch/srgssr/pillarbox/ui/widget/DelayedVisibilityState.kt

```diff
@@ -107,23 +114,23 @@ class DelayedVisibilityState internal constructor(
     @Suppress("UndocumentedPublicClass")
     companion object {
         /**
-         * Default duration
+         * Default auto-hide duration.
          */
         val DefaultDuration = 3.seconds
 
         /**
-         * Disabled duration
+         * Disabled auto-hide duration.
          */
         val DisabledDuration = ZERO
     }
 }
 
 /**
- * Toggleable
+ * Makes a Composable toggleable, controlling the visibility state of a [DelayedVisibilityState].
  *
- * @param enabled whether this toggleable will handle input events and appear enabled for semantics purposes
- * @param role the type of user interface element. Accessibility services might use this to describe the element or do customizations
- * @param delayedVisibilityState the delayed visibility state to link
+ * @param enabled Whether to handle input events and appear enabled for semantics purposes.
```

Comment:
Note sure _semantic_ suits well here. Is there a purpose related to a meaning here (really what I understand by _semantic_) or are you rather talking about some kind of overall behavior perceived externally?

Reply:
I've reused the wording used in Compose's `toggleable` `Modifier`: https://developer.android.com/reference/kotlin/androidx/compose/foundation/selection/package-summary#(androidx.compose.ui.Modifier).toggleable(kotlin.Boolean,kotlin.Boolean,androidx.compose.ui.semantics.Role,kotlin.Function1)
> whether or not this [toggleable](https://developer.android.com/reference/kotlin/androidx/compose/foundation/selection/package-summary#(androidx.compose.ui.Modifier).toggleable(kotlin.Boolean,kotlin.Boolean,androidx.compose.ui.semantics.Role,kotlin.Function1)) will handle input events and appear enabled for semantics purposes

It refers to the accessibility part of the component, ie. talkback (the screen reader) will indicate that the component is disabled.